pkg/network/node/networkpolicy.go

```diff
@@ -147,11 +147,14 @@ func (np *networkPolicyPlugin) initNamespaces() error {
 		npns.gotNamespace = true
 		np.namespacesByName[ns.Name] = npns
 
-		if vnid, err := np.vnids.WaitAndGetVNID(ns.Name); err == nil {
+		vnid, err := np.vnids.WaitAndGetVNID(ns.Name)
+		if err == nil {
```

Comment:
there's no need to rewrite this part; `err` is available in the `else` clause

Reply:
Ok, I removed it!